influxdb/server.sls

```diff
@@ -25,8 +25,17 @@ influxdb_service:
   service.running:
   - enable: true
   - name: {{ server.service }}
+  - init_delay: 5
```

Comment:
This is only needed if we run command after that point. So it can be in an if statement. Should I?